docs/sources/setup/install/helm/install-scalable/_index.md

```diff
@@ -46,40 +46,83 @@ It is not recommended to run scalable mode with `filesystem` storage.
    helm repo add grafana https://grafana.github.io/helm-charts
    ```
 
-1. Update the chart repository:
+2. Update the chart repository:
```

Comment:
Unless this was numbered incorrectly in the output, best practice is to number all steps as 1.

https://grafana.com/docs/writers-toolkit/write/markdown-guide/#numbered-lists